presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -693,4 +698,58 @@ public static long widthBucket(@SqlType(StandardTypes.DOUBLE) double operand, @S
 
         return lower;
     }
+
+    @Description("Cosine similarity between two Map<Varchar, Double>")
```

Comment:
Description should be English and not include the SQL type definition. Use words to explain it succinctly.